src/index.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 import GlobalStyles from './GlobalStyles.js';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
-  <React.StrictMode>
```

Comment:
I think you should leave Strict Mode in, I believe it gives additional checks / warnings.